docker-compose.numero.yml

```diff
@@ -44,8 +44,8 @@ services:
       retries: 10
 
   emulator:
-    image: gcr.io/flow-container-registry/emulator:0.27.3
-    command: emulator --persist
+    image: gcr.io/flow-container-registry/emulator:0.33.1
+    command: emulator --persist -v
```

Comment:
Just a note about this `-v` - this is the verbose flag, and I thought it was helpful while troubleshooting stuff sometimes. But it's also safe to remove if the logs become to chatty in the docker-compose setup.

Reply:
Oh nice - thanks for pointing this out